diy_set.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: page
+nav_order: 10
+---
+
+# SlimeVR DIY Kit
+{:.no_toc}
+
+## Thank you for purchasing the SlimeVR DIY kit!
+To get your DIY set operational, first it will need some additional attention. On this page we will guide you through the basic steps involved into creating your own ready-to-use set of SlimeVR trackers. While this guide contains some general recommendations, feel free to deviate from the norm and make them truly your own! Have fun!
+
+## Introduction
+### Box contents
+Inside your SlimeVR DIY kit are 7 large SlimeVR Mainboards, 3 small SlimeVR Tracker extensions, and 3 Extension cables (1 long, 2 short).
+
+TODO: Currently placeholder image
+![Box contents](./assets/img/Box_contents.png){: .caseImage}
+
+
+### Required extras
+To complete your SlimeVR trackers, you will need some parts in addition to your SlimeVR DIY kit. These include enclosures, straps, batteries, and a USB cable. These are detailed below to make it easier for you to complete your project.
+
+### Recommended mounting layout
+TODO: Currently placeholder image
+
+<table class="bpTable">
+   <tr>
+      <td>
+         <img id="bpImage" src="./assets/img/tracker_locations_7_3.png" alt="Mounting locations image"/>
+      </td>
+      <td>
+         <ol>
+            <li>Chest (Extension)</li>
+            <li>Waist</li>
+            <li>Upper legs</li>
+            <li>Lower legs</li>
+            <li>Feet (Extension)</li>
+            <li>Upper arms</li>
+         </ol>
+      </td>
+   </tr>
+</table>
+
+## Designing your case
+DIY kits require enclosures to protect them and allow strap attachment. These can be printed, purchased, or crafted.
+
+While the official SlimeVR Main and Extension cases are designed to fit the boards included in this kit, they are not optimised for home 3d printers due to their curved surfaces. As such, we recommend printing up your own using a modified design, or designing your own!
+
+For those planning to design your own enclosure, you can find the model of the Mainboard [here](LINK TO MAINBOARD STEP FILE LOCATION) and Extension PCBs [here](LINK TO EXT STEP FILE LOCATION).
```

Comment:
List of Cases like in DIY, but that host the Official PCB?

Reply:
So far the only case that I am aware off that supports the official PCB is the official case, which will be included as soon as i get the final files for the extensions. I will probably also make an edited version which is more optimized for FDM printers. But we can definitely add this if more cases become available?